cogs/induct.py

```diff
@@ -252,6 +238,19 @@ async def _perform_induction(self, ctx: TeXBotApplicationContext, induction_memb
                 reason=f"{ctx.user} used TeX Bot slash-command: \"/induct\"",
             )
 
+        intro_channel: discord.TextChannel | None = discord.utils.get(
+            self.bot.main_guild.text_channels,
+            name="introductions",
+        )
+
+        if intro_channel:
+            recent_message: discord.Message
+            for recent_message in await intro_channel.history(limit=30).flatten():
+                if recent_message.author.id == induction_member.id:
+                    await recent_message.add_reaction(":wave:")
+                    await recent_message.add_reaction(":TeX:")
+                    return
```

Comment:
I think the correct keyword you want here is `break` as that will exit the for-loop after reacting, but continue execution of the function (i.e. send the confirmation response below). Whereas `return` will completely exit the function, meaning anything after the for-loop will not be executed.